cargo-miri/bin.rs

```diff
@@ -882,7 +883,20 @@ fn phase_rustc(mut args: env::Args, phase: RustcPhase) {
 
     // Run it.
     if verbose {
-        eprintln!("[cargo-miri rustc] {:?}", cmd);
+        eprint!("[cargo-miri rustc] ");
+        let mut envs = HashMap::new();
+        for (key, value) in std::env::vars() {
+            envs.insert(key, value);
```

Comment:
oO you are printing *all* env vars? That seems a bit too much.^^

Printing `cmd.get_envs()` seems great, but I am not sure about all the ambient ones -- most of them will not be relevant for this.

Reply:
Can you make `-v` behave as before and only `-vv` be so extremely verbose?

Reply:
Also btw what if [env_remove](https://doc.rust-lang.org/nightly/std/process/struct.Command.html#method.env_remove) has been used? Will it still print the var here?

Reply:
I don't see an `envs.remove` anywhere.

Oh and can you make this a helper function, to avoid distracting from the actual purpose of `phase_rustc`?